openshift_scalability/ci/scripts/prometheus-loader.sh

```diff
@@ -9,22 +9,25 @@ enable_pbench=$5
 pbench_copy_results=$6
 pbench_user_benchmark=$7
 
-cd /root/svt/openshift_scalability
-# TODO: get the script file, remove this once the PR merged.
-wget https://raw.githubusercontent.com/mrsiano/svt/prometheus_loader/openshift_scalability/prometheus-loader.py
-wget https://raw.githubusercontent.com/mrsiano/svt/prometheus_loader/openshift_scalability/content/prometheus/qs.txt
+cd /root/svt/openshift_scalability # go to svt working dir
 
 oc login -u system:admin
 if [ "${enable_pbench}" == "true" ]; then
     pbench-register-tool-set
 fi
 # start the prometheus load.
-nohup python prometheus-loader.py -f ./qs.txt -i ${refresh_interval} -t ${concurrency} -p ${graph_period}  > /dev/null 2>&1 &
+nohup python prometheus-loader.py -f content/promethues/qs.txt -i ${refresh_interval} -t ${concurrency} -p ${graph_period}  > /dev/null 2>&1 &
 loader_pid=$(echo $!)
-# sleep 2.5 hours, and monitor the load by pbench.
-${pbench_user_benchmark} sleep ${duration}
+# sleep x hours, and monitor the load by pbench.
+${pbench_user_benchmark} sleep ${duration}; sleep 10; pbench-stop-tools
 # stop pbench and copy results.
 ${pbench_copy_results}
 # stop the promehteus load.
 kill -9 $loader_pid
+# dump logs
+oc logs prometheus-k8s-0 -c prometheus > ~/prometheus_oc_logs.log
```

Comment:
@sjug don't we have a $PBENCH_DIR or similar variable that can be used to store test logs and have them collected/archived by pbench automatically?

Reply:
Indeed, `${benchmark_run_dir}` see https://github.com/distributed-system-analysis/pbench/commit/1c52ee2155af6fd9f1a6d7a0ed7b82dab643a42a